### PR TITLE
updating for set quotacell status endpoint

### PR DESCRIPTION
**Pull request by SnehaCRao**

### CODE REVIEW

**Comment by SnehaCRao** [comment]
Adding new endpoint to LAUNCH/PAUSE quota cells:
{{api_host}}/sample/v1/projects/{{extProjectId}}/lineItems/{{extLineItemId}}/quotaCells/{{quotaCellId}}/pause
{{api_host}}/sample/v1/projects/{{extProjectId}}/lineItems/{{extLineItemId}}/quotaCells/{{quotaCellId}}/launch
This endpoint is yet to be added to developers.dynata.com

File: README.md [hunk 1]
```diff
@@ -73,7 +73,10 @@ Links to the Demand API documentation are included for each function.
 [Launch Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem-launch): launch_line_item(project_id, line_item_id)  
 [Pause Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem-pause): pause_line_item(project_id, line_item_id)  
 [Update Line Item](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/post-lineitem): update_line_item(project_id, line_item_id, line_item_data)  
-[Get Line Item Detailed Report](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/get-detailed-line-item): get_line_item_detailed_report(project_id, line_item_id)  
+[Get Line Item Detailed Report](https://developers.dynata.com/demand-api-reference/core-resources/lineitems/get-detailed-line-item): get_line_item_detailed_report(project_id, line_item_id) 
+
+### todo: will update this once it is published in developers.dynata.com
```
Comment (dynata-bradley) [inline README.md:78]: Not sure this note needs to be in the code. Why not denote it in Github and @DynataRidley will make sure to wait until the green light to release the next version to PyPi?

**Comment by dynata-bradley** [comment]
I don't think I can adequately review this without having the endpoint documentation.

File: README.md (hunk 1 above)
Comment (SnehaCRao) [reply README.md]: @DynataRidley , I have updated with the changes to README. Could you kindly review.

**Pull request merged**